### PR TITLE
User manual: makes minor grammar and style improvements in the sectio…

### PR DESCRIPTION
**Pull request by mdimjasevic**
In the user manual, this patch makes minor grammar and style improvements in the section on contributing.

### CODE REVIEW

File: doc/user-manual/contribute/index.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Contribute
 ##################
 
-See also the `HACKING` file in the root of the agda repo.
+See also the `HACKING` file in the root of the Agda repository.
```
Comment (andreasabel) [inline doc/user-manual/contribute/index.rst:7]: Approved! :)

File: doc/user-manual/contribute/documentation.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@ Rendering documentation locally
 
   To see the list of available targets, execute ``make help``
   in ``doc/user-manual``. E.g., call ``make html`` to build the
-  documentation in html format.
+  documentation in the html format.
```
Comment (andreasabel) [inline doc/user-manual/contribute/documentation.rst]: Not sure about this definitive article here. Confer:
a) "Dance this song in Gangnam style!"  
b) "Dance this song in the Gangnam style!"

My gut prefers a).

**Review by andreasabel — COMMENTED** [review]
Not sure about "in the html format".

File: doc/user-manual/contribute/documentation.rst (hunk 2 above)
Comment (mdimjasevic) [reply doc/user-manual/contribute/documentation.rst]: I'm not a native English speaker, but to me it does sound better with the article. Nevertheless, I might be wrong. I'll revert this change.

Comment (mdimjasevic) [reply doc/user-manual/contribute/documentation.rst]: I just force-pushed a commit without the "the html format" change.

**Comment by andreasabel** [comment]
Thanks!

**Pull request merged**